cmd/ferretdb/main.go

```diff
@@ -87,6 +87,7 @@ var cli struct {
 		RecordsDir string `default:"" help:"Testing: directory for record files."`
 
 		DisableFilterPushdown    bool `default:"false" help:"Experimental: disable filter pushdown."`
+		EnableSortPushdown       bool `default:"false" help:"Experimental: enable sort pushdown."`
```

Comment:
Why safe sort pushdown is not enabled by default, and why enabling it is considered experimental?